explorer/client/src/components/JobRuns/Details.tsx

```diff
@@ -125,6 +125,8 @@ interface IProps extends WithStyles<typeof styles> {
 const buildSearchQuery = (id: string) => `/job-runs?search=${id}`
 
 const Details = ({ classes, jobRun, etherscanHost }: IProps) => {
+  const nodeHasUrl = jobRun.chainlinkNode.url !== ''
```

Comment:
How do you feel about just using the truthiness for this? e.g. `{jobRun.chainlinkNode.url  ? ... : ... }`

Reply:
Sure, thanks.